e2e/interchaintestv8/testvalues/values.go

```diff
@@ -87,7 +89,7 @@ const (
 	DefaultGovV1ProposalTokenAmount = 500_000_000
 
 	// IbcCommitmentSlotHex is the storage slot in the IBC solidity contract for the IBC commitments.
-	IbcCommitmentSlotHex = "0x1"
+	IbcCommitmentSlotHex = ics26router.IbcStoreStorageSlot
```

Comment:
New commitment slot. It is an issue that forge inspect can't see ERC-7201 storage slots at all. I found this by guessing (educated)